instrumentation/kamon-redis/src/main/scala/kamon/instrumentation/jedis/JedisInstrumentation.scala

```diff
@@ -14,36 +14,47 @@
  * limitations under the License.
  */
 
-package kamon.instrumentation.jedis
+package kamon
+package instrumentation
+package jedis
 
 import kamon.Kamon
 import kamon.trace.Span
 import kanela.agent.api.instrumentation.InstrumentationBuilder
 import kanela.agent.libs.net.bytebuddy.asm.Advice
 import kanela.agent.libs.net.bytebuddy.description.method.MethodDescription
-import kanela.agent.libs.net.bytebuddy.matcher.ElementMatchers.isPublic
+import kanela.agent.libs.net.bytebuddy.matcher.ElementMatchers.{isMethod, isPublic, namedOneOf, not, whereAny}
+import redis.clients.jedis.Protocol
 
 class JedisInstrumentation extends InstrumentationBuilder {
-  onType("redis.clients.jedis.Protocol")
-    .advise(method("sendCommand").and(isPublic[MethodDescription]), classOf[SendCommandAdvice])
+  onType("redis.clients.jedis.Jedis")
```

Comment:
`redis.clients.jedis.Jedis` is only one of the implementations of `redis.clients.jedis.JedisCommands`. I guess we should add tests for the other classes...

Reply:
I noticed that there is `Jedis` and `ShardedJedis`, but the sharded version uses a regular Jedis under the hood so it should work fine. There are other variants too, like `JedisCluster` and in 4.x there is a `JedisPool` too. I'll leave all of those for a separate PR.

Reply:
More details here: https://github.com/kamon-io/Kamon/issues/1121